### DISCUSSION TITLE
Graph Interval

### DISCUSSION THREAD

**Discussion by emmanmarquez** [akvorado/akvorado#1058] [Q&A]
Is there a way to change the graph interval? The graph shows every 1 minute. I have a configuration on akvorado.yaml and I've tried for 10 minutes and 30 minutes but both configs are not working. Am I on the right page to do the configuration?

![image](https://github.com/akvorado/akvorado/assets/123536938/ac69254a-3474-4633-a793-72f5b6a3cfe6)
  
  
![image](https://github.com/akvorado/akvorado/assets/123536938/dc758dd6-b2e2-4a53-890c-eddd4fa6efd7)
![image](https://github.com/akvorado/akvorado/assets/123536938/667666d5-d9dd-4428-aefa-90d0921ce800)

**Comment by vincentbernat** [comment]
No, the "best" resolutions is selected automatically and there is little to do to alter that. You could zoom out and hope you'll get a 30 minute interval. Also, I am not even sure how your configuration could work. Start from the default configuration for tweaks.

**Reply by emmanmarquez** [reply]
Hi Vincent, may I verify if that configuration is for data retention? As far as I can understand, that configuration with resolution parameters is for the data retention stored in docker-akvorado-clickhouse. We added some config because disk space is continuously growing. My VM disk has only 500GB to cater for storing data.

As far as I understand that config, let me break it down:

Resolutions:

-   interval: 5m #if this is set to 5 minutes, then the interval for getting the data is every 5 minutes and storing it in the database or wherever it is stored. the longer the minutes the smaller the data has been saved
  
- TTL: 168h #  how long to keep the data in the database. if it reaches the target hours, then it will be removed from the database

- system-log-ttl: 10080m # System log tables. If they reach the set config, it will also delete the stored log, whatever tables it has stored

**Reply by vincentbernat** [reply]
Interval is used for consolidation. Data is ingested continuously and is consolidated in buckets of 5 minutes.

**Reply by emmanmarquez** [reply]
Thanks, Vince for your clarification, can you provide some retention examples?

**Reply by vincentbernat** [reply]
Check the default configuration, which is also reproduced in the documentation.

**Reply by emmanmarquez** [reply]
What I mean is can you provide the retention examples for keeping only 14 days of data? In my current configuration, I used this 

`resolutions:
    - interval: 10m
    - TTL: 336h 
 system-log-ttl: 20160m`
 
based on your reply, I did not see any clarification that my retention config is for keeping the data for 14 days, and If my current config is not for that, can you please instruct or give me examples for my disk not growing continuously because I can cater only 500GB of stored data

**Reply by vincentbernat** [reply]
15 days is `360h`. Again, modify the default configuration, you can't just have `interval: 10m`. The default configuration is in the [documentation](https://demo.akvorado.net/docs/configuration#clickhouse). There is also a [query](https://demo.akvorado.net/docs/operations#space-usage) to see how much each table is using. Space is usually taken by the main table. Consolidated tables are usually smaller, even with a longer retention.
